tests/ForeignRelationTest.php

```diff
@@ -19,5 +18,14 @@ public function testAbilityToCreateRelationForSingleField()
 		$relation = ForeignRelationship::fromString("name:fooModel;is-nullable:true;data-type:varchar;foreign-relation:assets#hasMany#App\\Models\\Asset|category_id|id");
 		
 		// TO DO, asset that the relation is created successfully!
+        $this->assertTrue($relation instanceof ForeignRelationship);
```

Comment:
Simplest and most basic step I could think of towards Fearless Leader's TODO.  If what you get isn't a ForeignRelationship, then it definitely hasn't been created successfully.

@malhayek2014 , what conditions have to be satisfied for the relation to be created successfully?

Reply:
That could have been phrased better.  Are there any cases where this could generate a ForeignRelationship that is broken or wrong?